cumulus/parachains/runtimes/starters/seedling/src/lib.rs

```diff
@@ -354,8 +354,11 @@ impl_runtime_apis! {
 	}
 
 	impl sp_session::SessionKeys<Block> for Runtime {
-		fn generate_session_keys(seed: Option<Vec<u8>>) -> Vec<u8> {
-			SessionKeys::generate(seed)
+		fn generate_session_keys(_: Vec<u8>, _: Option<Vec<u8>>) -> sp_session::OpaqueGeneratedSessionKeys {
+			sp_session::OpaqueGeneratedSessionKeys {
```

Comment:
Nitpick. Makes sense to implement `Default` for `OpaqueGeneratedSessionKeys`? I see a couple of use cases around